src/overlays/actors/ovl_Obj_Tokei_Turret/z_obj_tokei_turret.c

```diff
@@ -28,23 +28,67 @@ const ActorInit Obj_Tokei_Turret_InitVars = {
     (ActorFunc)ObjTokeiTurret_Draw,
 };
 
-// static InitChainEntry sInitChain[] = {
-static InitChainEntry D_80B91EC0[] = {
+static InitChainEntry sInitChain[] = {
     ICHAIN_F32(uncullZoneForward, 1200, ICHAIN_CONTINUE),
     ICHAIN_VEC3F_DIV1000(scale, 100, ICHAIN_STOP),
 };
 
-#endif
+void ObjTokeiTurret_Init(Actor* thisx, GlobalContext* globalCtx) {
+    ObjTokeiTurret* this = THIS;
+    Actor* actor;
+    s32 actorParams;
 
-extern InitChainEntry D_80B91EC0[];
+    actor = &this->dyna.actor;
+    actorParams = actor->params & 3;
+    this->dyna.actor = *actor;
```

Comment:
This is a very strange construction, are you sure it's doing this, rather than just m2c having delusions?

Reply:
Indeed, I got curious, you don't need the actor temp at all: https://decomp.me/scratch/qwplC